hw-10/main.js

```diff
@@ -0,0 +1,117 @@
+const printMultiplication = number => {
+  if (number < 1 || number > 10) return;
+  console.log('in bang nhan')
+  for (let i = 0; i <= 10; i++) {
+    console.log(`${number} x ${i} = ${number * i}`)
+  }
+}
+//b2 in cac số chẵn
```

Comment:
```suggestion
// b2 in các số chẵn
```
Và cũng ko được ghi tiếng Việt, tất cả mọi thứ đều phải dùng tiếng Anh ngoại trừ được yêu cầu dùng ngôn ngữ khác hoặc bắt buộc phải dùng ngôn ngữ khác